spec/lib/simple_forms_api_submission/metadata_validator_spec.rb

```diff
@@ -145,7 +145,7 @@
         }
         expected_metadata = {
           'veteranFirstName' => 'John',
-          'veteranLastName' => 'John - John',
+          'veteranLastName' => 'John - Jo/hn',
```

Comment:
Do you have a test that includes an apostrophe that is stripped form a name? That was the character that triggered most, if not all, of the validation failures we encountered for form 4142.

Reply:
I can add `’` and `'` to the test. 👍